sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogDetailLevel.java

```diff
@@ -36,10 +37,17 @@ public enum HttpLogDetailLevel {
     /**
      * Logs everything in HEADERS and BODY.
      */
-    BODY_AND_HEADERS;
+    BODY_AND_HEADERS,
+
+    /**
+     * Logs everything in BASIC, plus allowed request and response headers.
+     */
+    ALLOWED_HEADERS;
 
     static final String BASIC_VALUE = "basic";
     static final String HEADERS_VALUE = "headers";
+    static final String ALLOWED_HEADERS_VALUE = "allowed_headers";
+    static final String ALLOWEDHEADERS_VALUE = "allowedheaders";
```

Comment:
Why do we have two constants with similar names?

Reply:
no longer applies, but we do the same for `body_and_headers` and `bodyandheaders`